src/domain/repositories/IndicatorsRepository.ts

```diff
@@ -0,0 +1,52 @@
+import { Id } from "domain/entities/Base";
+
+export interface IndicatorsRepository {
+    get(ids: Id[]): Promise<Indicator[]>;
+    getDataElementsNames(ids: Id[]): Promise<metadataItemName[]>;
+    getCOCombosNames(ids: Id[]): Promise<metadataItemName[]>;
+    exportIndicatorsDataToCSV(metadata: indicatorDataRow[], path?: string): Promise<void>;
+    exportValuesReportToCSV(metadata: valueReportRow[], path?: string): Promise<void>;
+}
```

Comment:
This repo is mixing a lot of different things: The first one is ok; the other two get's returns things which are not indicators, they do not belong here. The two export ones (which should not include the word "csv", that's an implementation detail of how the report will be exported), should be in separate repositories. Note that the entity and the repo should match, for example: `interface IndicatorsReportRepository { export(report: IndicatorsReport): Promise<void>; }`

Reply:
Created DataElements and COCombos repos for the get*Names.
Not sure about needing a repo for the reports.

Reply:
Ok, let's keep the reports as-is.